test/e2e/storage/utils/create.go

```diff
@@ -22,6 +22,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"html/template"
```

Comment:
I can see other parts use `text/template`, is there a specific reason for html?

Reply:
I think it was an automatic import by my editor...